docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,14 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes updated versions of filters that will not run on JDK 8.
+We are actively patching the build scripts for all filters that are affected. 
```

Comment:
Does it make sense to reference the actual problem filters? 

`logstash-filter-date` and
`logstash-filter-geoip`? And refer to the [update plugins](https://www.elastic.co/guide/en/logstash/current/working-with-plugins.html#updating-plugins) section on how to do this?